content/200-concepts/100-components/07-preview-features/100-sql-server/index.mdx

```diff
@@ -61,9 +67,76 @@ To make local development easier on MacOS, you can add `encrypt=DANGER_PLAINTEXT
 
 > **Note**: `encrypt=DANGER_PLAINTEXT` must not be used in production.
 
-###  Migrations are not supported yet
+###  Prisma Migrate (Preview) caveats
+
+Prisma Migrate (Preview) is supported in 2.13.0 and later with the following caveats:
+
+#### Native types
+
+Prisma Migrate (Preview) does not yet support native types, which means that you cannot use SQL Server, Prisma Migrate, and native types together.
+
+
+#### Database schema names
+
+SQL Server does not have an equivalent to the PostgreSQL `SET search_path` command familiar from PostgreSQL. This means that when you create migrations, you must define the same schema name in the connection string that’s being used on production. For most of the users this is `dbo` (the default value). But if the production database is using some other schema name, all the migration SQL must be either edited by hand to reflect the production, or in connection string before creating any migrations (defined as `schema=name`).
+
+#### Cascading rules
+
+We made the decision to follow the same cascading rules as we do in the other databases. It means for most of the relations (with certain restrictions), we get `ON DELETE CASCADE ON UPDATE CASCADE`. This is a good default for how simple it is to delete all children from a parent automatically, but with SQL Server the cascading has certain restrictions that do not allow us to create certain kinds of relations.
+
+First we’ll see a working schema:
+
+```java
+model Album {
+  id        Int    @id @default(autoincrement())
+  artist_id Int
+  artist    Artist @relation(fields: [artist_id], references: [id])
+}
+
+model Artist {
+  id     Int      @id @default(autoincrement())
+  albums Album[]
+}
+```
+
+Now we want to add the favorite album to every artist:
+
+```glsl
```

Comment:
Why java and glsl? and not `prisma`

Reply:
These are copied from my notion document, which doesn't support Prisma highlighting.